bin/craft

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Craft console bootstrap file
+ */
+
+if (!empty($_composer_autoload_path)) {
+    $basePath = dirname($_composer_autoload_path) . '/../';
+}
+else {
+    $basePath = getcwd();
+}
+
+// Define path constants
+define('CRAFT_BASE_PATH', $basePath);
+define('CRAFT_VENDOR_PATH', CRAFT_BASE_PATH . '/vendor');
+
+// Load Composer's autoloader
+require_once CRAFT_VENDOR_PATH . '/autoload.php';
+
+// Load dotenv?
+if (class_exists(Dotenv\Dotenv::class)) {
+    // By default, this will allow .env file values to override environment variables
+    // with matching names. Use `createUnsafeImmutable` to disable this.
+    Dotenv\Dotenv::createUnsafeMutable(CRAFT_BASE_PATH)->safeLoad();
```

Comment:
`createUnsafeMutable()` was added in PHP Dotenv 5.0, but several Craft installs have older versions. Maybe we should use some reflection here and call `createMutable()` if `createUnsafeMutable()` doesn’t exist, or `create()` if `createMutable()` doesn’t exist?

Reply:
Yea, the conditionals to get this working are pure mayhem.

https://github.com/markhuot/craft-pest/blob/35b45fa8afa097667579a41b1cdb1663ffcceb28/src/test/TestCase.php#L60-L69

I don't know if it'd be better to read the `composer.json` so you know exactly what version is being used? I haven't dug in too deeply here but happy to over the next few days.